server/portal/apps/request_access/api/views.py

```diff
@@ -0,0 +1,65 @@
+from django.views.decorators.csrf import ensure_csrf_cookie
+import logging
+from django.http import JsonResponse, HttpResponseBadRequest
+from django.conf import settings
+from portal.apps.tickets import rtUtil
+from portal.views.base import BaseApiView
+from pytas.http import TASClient
+
+logger = logging.getLogger(__name__)
+
+METADATA_HEADER = "*** Ticket Metadata ***"
+
+class RequestAccessView(BaseApiView):
+    def post(self, request):
+        """Post an access request
```

Comment:
Since this is mostly the same ticket submission code from `server/portal/apps/tickets/api/views.py`, can we find a way to use the same code, and wrap it with the authentication portion of this file?

Reply:
Yeah, definitely.  Very important to minimize the duplicate code to make maintaining easier. Especially regarding the all the metadata that gets stuck to a ticket as we would only want to maintain similar lines of code doing that.

Can a unit test(s) also be added to this route?